src/plugins/data-card-tool/data-card-tool.tsx

```diff
@@ -81,14 +81,35 @@ export const DataCardToolComponent: React.FC<IToolTileProps> = observer((props)
   }
 
   function deleteCase(){
-    // TODO modal (see src/components/delete-button)
     const thisCaseId = content.dataSet.caseIDFromIndex(content.caseIndex);
     if (thisCaseId) {
       content.dataSet.removeCases([thisCaseId]);
     }
     previousCase();
   }
 
+  const AlertContent = () => {
+    return <p>Remove the current Data Card?</p>;
```

Comment:
```suggestion
    return <p>Delete the current Data Card?</p>;
```
Unless the spec says otherwise, it seems like we should be consistent in our wording, either using "Delete" or "Remove" throughout.